src/frontend/src/binder/expr/value.rs

```diff
@@ -113,12 +113,37 @@ impl Binder {
 
         Ok(literal)
     }
+
+    /// `ARRAY[...]` is represented as an function call at the binder stage.
+    pub(super) fn bind_array(
+        &mut self,
+        exprs: Vec<risingwave_sqlparser::ast::Expr>,
```

Comment:
nit: what about importing it? It will not be ambiguous as all parameters of `bind_xxx` can be assumed to be from `ast`.

Reply:
Actually, it's for me not to be confused, Every time I come across a Expr will wonder whether it's a ast Expr or crate::Expr. (:smilecry

Sure I will directly import in at least in this case.